README.md

```diff
@@ -1,41 +1,112 @@
+# Migrating receipts to RevenueCat via bulk import
+
 For large datasets that aren't practical to import through the REST API, you can send us a `.csv` file to import manually. This repository contains sample `csv` files that we can import. [Email us](mailto:support@revenuecat.com) with the below information and we'll be in touch.
 
 > NOTE: Bulk imports DO NOT trigger webhooks or integrations. Bulk imports done by RevenueCat will not trigger any webhook or integration events. If this is a requirement, then you'll need to perform an import using the REST API as mentioned in the [RevenueCat Migration Docs](https://docs.revenuecat.com/docs/migrating-existing-subscriptions#using-the-rest-api). 
 
-To do a receipt import we'll need 3 csv files for iOS, Android and Stripe with the following fields:
+## Receipt Import CSV File Format
+
+To do a receipt import we'll need 3 csv files for iOS, Android and Stripe with the following fields (✅ = required for that platform/store):
+
+| Column Name                   | iOS | Android | Stripe |
+|-------------------------------|-----|---------|--------|
+| `app_user_id`                 | ✅   | ✅       | ✅      |
+| `receipt`/`token`             | ✅   | ✅       | ✅      |
+| `product_id`                  | ✅   | ✅       |        |
+| `price`                       | ✅   |         |        |
+| `currency`                    | ✅   |         |        |
+| `introductory_price`          | ✅   |         |        |
+| `introductory_price_duration` | ✅   |         |        |
+
+### iOS
+
+> 💡 Check out the [sample file](iOS/ios_product_price_map_sample.csv) for a complete example of a product price map CSV file for iOS.
+
+`app_user_id` The user identifier to associate with the receipt.
+
+`receipt` The raw receipt file from the device.*
+
+`product_id` The last purchased product ID contained in the receipt. This is for pricing calculations - if you don't have the product ID, you need to provide a product price map file.
+
+`price` The price of the product.
+
+`introductory_price` The introductory offer price. This column is required if you offer an introductory period that is not a free trial.
+
+`currency` The currency of the price. If you don't have prices and currencies, you need to provide a product price map file.
+
+> If you don't have product IDs, prices or currencies with the receipts we can take a [separate file](#product-price-map-ios-only) with a mapping of product_id, price, currency as well. If there were any known price changes, those should also be included here.
+
+***Note: The latest_receipt_info is not supported as it's only a subset of the entire receipt.** The entire iOS receipt file must be used for imports.
+
+#### Product Price Map (iOS Only)
```

Comment:
This whole product price map section is new, 👀 appreciated!

Reply:
Do you think we should add an 🚧  icon or something next to some of the iOS fields that are "recommended" but not required? We could call out in the description what the implications are if this field is not provided. I know a lot of developers don't store `price` for example and need to do the Product Price Map - we could add that to the description of the `price` attribute ("If this column is not provided, a product price map is required and USD price will be assumed for all transactions")

Reply:
I originally had something like that but then removed it because I wasn't sure if it would be too wordy. I'll add it back in!

Reply:
Changes [here](https://github.com/RevenueCat-Samples/import-csv-samples/pull/4/commits/75e2ee97140a4e9e6593cf27092eaf49be5c0951) and [here](https://github.com/RevenueCat-Samples/import-csv-samples/pull/4/commits/143ac4e5f6b46d26836b1bda2e3089b0d7d15080)